shoop/order_printouts/locale/en/LC_MESSAGES/django.po

```diff
@@ -38,6 +38,19 @@ msgstr ""
 msgid "Order confirmation"
 msgstr ""
 
+#, python-format
+msgid "Possible file system traversal shenanigan detected with %(path)s"
+msgstr ""
+
+#, python-format
+msgid "Unable to find: %(path)s"
+msgstr ""
```

Comment:
ValueError messages should not be marked for translation (see prev comment for details).